content/cpp/concepts/math-functions/terms/cosh/cosh.md

```diff
@@ -11,17 +11,25 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-Returns the hyperbolic cosine of an argument given in radians.
+The **`cosh()`** function returns the hyperbolic cosine of an argument given in radians.
 
 ## Syntax
 
 ```cpp
 std::cosh(n)
 ```
+where `n` is the angle in `radians`
```

Comment:
```suggestion
Where `n` is the angle in `radians`.
```

Reply:
@kitrak-rev We still need to capitalize this word.

Reply:
Actually, could we consider this suggestion?

```suggestion
The `angle` is measured in radians.
```